ambry-router/src/main/java/com/github/ambry/router/PutBlobMetaInfo.java

```diff
@@ -127,7 +127,10 @@ public static PutBlobMetaInfo deserialize(String metaInfo) throws IOException {
     }
 
     jsonNode = rootNode.get(RESERVED_METADATA_CHUNK_ID);
-    String metaBlobId = jsonNode.textValue();
+    String metaBlobId = null;
+    if (jsonNode != null) {
+      metaBlobId = jsonNode.textValue();
+    }
```

Comment:
Adding comment here since it doesn't allow me to add at the top of this class :) Would it be helpful to add a 'json example' at the top of this class?